src/OpenAI-DotNet-Proxy/Proxy/OpenAIProxyStartup.cs

```diff
@@ -18,7 +18,7 @@ namespace OpenAI.Proxy
     /// <summary>
     /// Used in ASP.NET Core WebApps to start your own OpenAI web api proxy.
     /// </summary>
-    public class OpenAIProxyStartup
+    public sealed class OpenAIProxyStartup
```

Comment:
I actually thought about this. It can stay unsealed uncase someone wants to extend this class